warehouse/query-core/src/test/java/datawave/query/config/ShardQueryConfigurationTest.java

```diff
@@ -0,0 +1,78 @@
+package datawave.query.config;
+
+import com.google.common.collect.*;
+import datawave.data.type.DateType;
+import datawave.data.type.StringType;
+import datawave.data.type.Type;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Map;
+import java.util.Set;
+
+public class ShardQueryConfigurationTest {
+    
+    private ShardQueryConfiguration config;
+    
+    @Before
+    public void setUp() {
+        config = ShardQueryConfiguration.create();
+    }
+    
```

Comment:
I know you are going to groan on this one, but is there a way to verify that the BeanUtils copied everything?  Also does the Bean Utils create deep copies or are all of the collections copied by reference as that could be a problem.

Reply:
I'll work something up to verify a full copy. The decompiled bytecode indicates a deep copy is occurring, but I'll also add in a test for that.